### ISSUE TITLE
A11y gaps in form controls and uptime dashboard widget

### ISSUE THREAD

**Issue by seanforyou23** [syndesisio/syndesis#3075]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## The problem

Several form controls in the application do not have accessible names attached. Also the dashboard metric "uptime" needs some tweaking on the markup side of things. The heading is H3, unlike similar cards being H2's, also the date used here shouldn't be a heading, but rather a `<time>` element maybe?  

## Expected behavior

All interactive controls should have an accessible name of some kind, so that users of assistive technology can understand what a given control without having to first "try it out".

## Screenshot

I haven't found a way to take screenshots with VoiceOver application on and router menu active, so I've went old school and taken a photo with a phone that illustrates the problem.

![20180713_142212](https://user-images.githubusercontent.com/5942899/42707930-ae794ad2-86a9-11e8-8d0f-5c86279e8540.jpg)
![20180713_142310](https://user-images.githubusercontent.com/5942899/42707932-ae909b2e-86a9-11e8-9e3d-22a9311b1071.jpg)
![20180713_142330](https://user-images.githubusercontent.com/5942899/42707933-aea2acb0-86a9-11e8-82a5-f593777fe314.jpg)

## Tasks involved / Steps to Reproduce

1. Launch VoiceOver (mac) with CMD + F5
2. Navigate to one of many views within the app and launch rotor menu with VO + U
3. Notice the missing accessible names, filled with a "?" and other inadequate text like "button"
4. Some of these fixes may need to be applied on the patternfly-ng side of things, as we're simply consuming some of these controls as part of larger components from that framework.
